packages/cli/lib/index.js

```diff
@@ -92,4 +93,20 @@ prog
 	.option('-p, --port', 'Set server port', 8080)
 	.action(commands.watch);
 
+prog
+	.command('info')
+	.describe('Print out debugging information about the local environment')
+	.action(() => {
+		console.log();
+		console.log('Environment Info:');
+		envinfo
+			.run({
+				System: ['OS', 'CPU'],
+				Binaries: ['Node', 'Yarn', 'npm'],
+				Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+				npmGlobalPackages: ['preact-cli'],
```

Comment:
good to merge if you also add the local versions of `preact`, `preact-compat` and `preact-cli` (myb `preact-router` and `preact-render-to-string`

Reply:
:+1: